eclass/distutils-r1.eclass

```diff
@@ -2004,6 +2058,44 @@ distutils-r1_src_configure() {
 	return ${ret}
 }
 
+# @FUNCTION: _distutils-r1_compare_installed_files
+# @INTERNAL
+# @DESCRIPTION:
+# Verify the the match between files installed between this and previous
+# implementation.
+_distutils-r1_compare_installed_files() {
+	debug-print-function ${FUNCNAME} "${@}"
+
+	# QA check requires diff(1).
+	if ! type -P diff &>/dev/null; then
+		return
+	fi
```

Comment:
How common is this case? I'm asking since I'm interested if we want to notify of the skip or not? For example, so we have in build.log from tinderbox that "sorry, we skipped the check" in case weird stuff happens? Or maybe it is rare to not have and we can just ignore me?

Reply:
`diffutils` are part of `@system`, so rather rare. However, I suppose they could be missing during bootstrap.